script/DeployCreatorTech.s.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.22;
+
+import {Script, console2} from "forge-std/Script.sol";
+import {CreatorTech} from "../src/CreatorTech.sol";
+
+contract DeployScript is Script {
```

Comment:
In reality, Hologram will probably have a single signer key. Perhaps we can have this be an input argument in the run() function below

Reply:
done